app/models/validation_request.rb

```diff
@@ -260,9 +260,8 @@ def time_extension_request?
     type == "TimeExtensionValidationRequest"
   end
 
-  def update_planning_application!(params)
-    # Specific types of validation request use this method, which is overwritten in those models.
-    #  A couple don't hence the empty method
+  def update_planning_application!(*)
+    # Subclasses must implement any necessary updates
```

Comment:
The only place that is passing params here is a legacy v1 API call for ownership certificate updates - which should be just bops-applicants so once the merge is final we can remove a lot of complexity.